api/helpers.go

```diff
@@ -28,65 +22,6 @@ func (mr *malformedRequest) Error() string {
 	return mr.msg
 }
 
-func decodeJSONBody(w http.ResponseWriter, r *http.Request, dst interface{}) error {
```

Comment:
not sure we use this anywhere so I removed it

Reply:
did a quick search and don't see that func being called anywhere. So you're right.